sdk/communication/azure-communication-chat/azure/communication/chat/_shared/user_credential_async.py

```diff
@@ -86,6 +92,13 @@ def _token_expiring(self):
     def _is_currenttoken_valid(self):
         return self._get_utc_now() < self._token.expires_on
 
+    def _convert_expires_on_datetime_to_utc_int(self):
```

Comment:
what if we add this method into the utils.py in this folder, and make this conversion during the token creation? Would that work? If that is the case you would not need to call this conversion method every time you return, as the token will be created with the int `expires_on` field